### ISSUE TITLE
Failed to instantiate provider v2.0.2

### ISSUE THREAD

**Issue by amanda-skilljar** [terraform-providers/terraform-provider-heroku#219]
### Terraform Version

```
Terraform v0.12.4
```

### Heroku Provider Version

```
+ provider.heroku v2.0.2
```

### Affected Resource(s)

N/A

### Terraform Configuration Files

```hcl
provider "heroku" {
  email   = var.heroku_email
  api_key = var.heroku_api_key
  version = "~> 2.0.2"
}
```

### Debug Output

Available upon request.

### Expected Behavior

`terraform plan|apply` should succeed.

### Actual Behavior

```bash
Error: Failed to instantiate provider "heroku" to obtain schema: Incompatible API version with plugin. Plugin version: 4, Client versions: [5]
```

### Steps to Reproduce

1. Run `terraform plan` or `terraform apply`.

### Important Factoids

N/A

### References

N/A

**Comment by mars** [comment]
Hi @amanda-skilljar , thanks for posting this issue.

It looks like something broke in the v2.0.2 release this morning, despite all of our tests passing.

You can use pin to the previous release as a workaround, until we fix it:

```
provider "heroku" {
  version = "2.0.1"
}
```

**Comment by davidji99** [comment]
Sorry about that @amanda-skilljar. We are fixing this.

**Comment by mars** [comment]
We released [v2.0.3](https://github.com/terraform-providers/terraform-provider-heroku/releases/tag/v2.0.3) with the fix.

**Comment by amanda-skilljar** [comment]
Thank you!
